tools/mqtt-bench/bench.go

```diff
@@ -14,33 +14,42 @@ import (
 	"strconv"
 	"time"
 
+	mf_log "github.com/mainflux/mainflux/logger"
```

Comment:
@dborovcanin do we typically use underscores in import names?

Reply:
We have used mflog at other places, I should've checked this 😅

Reply:
I've updated the PR, so it's also ready for review @drasko @dborovcanin

Reply:
We don't, good catch.